next.config.js

```diff
@@ -34,6 +34,11 @@ const nextConfig = {
         protocol: "https",
         hostname: "profile.stage.mozaws.net",
       },
+      // Stores breached companies' logos
```

Comment:
```suggestion
      // Stores breached companies logos
```

Reply:
I think this is correct? The logos are from the breached companies, so it's a possessive (alternative: "breached companies's logos").

Reply:
Oh, might be — sorry I though it just sneaked its way in.